### ISSUE TITLE
Combo boxes are not associated with their labels

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/instructor-tools#654]
Based on the [Accessibility Review Report](https://docs.google.com/spreadsheets/d/1vt4uJuSq3itNu8Ouq6KI-4KOdOlOi_T9PoqOVy0e3T0):

Severity: 4- Critical

Test: All pages

Issue notes: Combo boxes are not associated with their labels.

Category: Semantics and Structure

Criterion: [1.3.1 – Info and Relationships](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#info-and-relationships)

Impact: People using any type of assistive technology will have barriers or difficulty navigating or accessing this information.

Most affects people who are/have: Blind, Low vision, Neurodivergent
